src/libraries/Microsoft.PowerFx.Core/Syntax/Visitors/FindNodeVisitor.cs

```diff
@@ -270,18 +270,18 @@ public override bool PreVisit(TableNode node)
             }
 
             // Cursor is between the open and closed bracket.
-            for (var i = 0; i < node.Children.Length; i++)
+            for (var i = 0; i < node.Commas.Length; i++)
             {
                 // Cursor position is inside ith child.
-                if (_cursorPosition <= node.Children[i].Token.Span.Lim)
+                if (_cursorPosition <= node.Commas[i].Span.Lim)
                 {
                     node.Children[i].Accept(this);
                     return false;
                 }
             }
 
-            // If we got here the cursor is not within the brackets. return tableNode.
-            _result = node;
+            // If we got here the cursor should be in the last child.
+            node.Children[node.Children.Length - 1].Accept(this);
```

Comment:
>node.Children.Length  [](http://example.com/codeflow?start=26&length=21)

Is this possible to be 0 length?

Reply:
No, the previous code block checks for 3 edge cases:

Cursor before the [
Empty (0 children)
Cursor after the ]